CHARTER.md

```diff
@@ -131,8 +132,8 @@ Additional Chairs may be added so long as the existing number of Chairs is odd.
 In the event that an even number of Chairs exist and vote situation arises, the Chairs will randomly select one chair to abstain. 
 
 ### Role of Sub-Working Groups 
-Each Sub-Working Group (SWG) must have a Chair working as an active sponsor. Under the mandate of the Working Group, each SWG will have the autonomy to establish their own charter, membership rules, meeting times, make in-scope decisions as delegated, and management processes.
+Each Sub-Working Group (SWG) must have a Chair working as an active sponsor. Under the mandate of the Working Group, each SWG will have the autonomy to establish their own charter, membership rules, meeting times, and management processes. Each SWG will also have the authority to make in-scope decisions as delegated by the Working Group.
 
-SWG are required to submit their agreed Charter to the Working Group for information and archival. The Chairs can petition for dissolution of an inactive or hostile SWG by Call for Agreement. Once dissolved the SWG’s delegated Charter and outstanding authority to make decisions is immediately revoked. The Chairs may then take any required action to restrict access to Working Group Resources.  
+SWG are required to submit their agreed Charter to the Working Group for information and archival. The Chairs can petition for dissolution of an inactive or hostile SWG by Call for Agreement. Once dissolved the SWG’s delegated Charter and outstanding authority to make decisions is immediately revoked. The Chairs may then take any required action to restrict access to Working Group Resources.
```

Comment:
Should the first use of `SWG` be plural?

Reply:
yes, thanks!